c/CMakeLists.txt

```
@@ -71,6 +78,9 @@ set(STATIC_LIBRARY_INSTALL_PATH "${BINARY_INSTALL_PATH}/${HYPERONC_STATIC_LIB_FI
 include(CMakePackageConfigHelpers)
 
 if(BUILD_SHARED_LIBS)
+    find_package(OpenSSL REQUIRED)
+    find_package(ZLIB REQUIRED)
```

Should be looked only when `GIT` option is `ON`
```suggestion
    if(GIT)
        find_package(OpenSSL REQUIRED)
        find_package(ZLIB REQUIRED)
    endif()

```